assignments/objects.js

```diff
@@ -18,27 +18,83 @@ const example = {
 }
 
 // Write your intern objects here:
+const intern1 = {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+}
+
+const intern2 = {
+  "id": 2,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M"
+}
+
+
+const intern3 = {
+  "id": 3,
+  "name": "Kevin",
+  "email": "kmummery2@wikimedia.org",
+  "gender": "M"
+}
+
+const intern4 = {
+  "id": 4,
+  "name": "Gannie",
+  "email": "gmartinson3@illinois.edu",
+  "gender": "M"
+}
+
+const intern5 = {
+  "id": 5,
+  "name": "Antonietta",
+  "email": "adaine5@samsung.com",
+  "gender": "F"
+}
+
 
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
 
+console.log(intern1["name"]);
+
 // Kennan's ID
 
+console.log(intern2["id"]);
+
 // Keven's email
 
+console.log(intern3["email"]);
+
 // Gannie's name
 
+console.log(intern4["name"]);
+
 // Antonietta's Gender
 
+console.log(intern5["gender"]);
+
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
-// console.log(kennan.speak());
+
+ function kennan (name) { 
+   console.log("Hello, my name is " + name + "!");
+ }
+  kennan(intern2["name"]);
+
```

Comment:
An interesting approach, but you could simplify this even further by adding the function to the kennan object itself. For instance:
```js
kennan.speak = function() {
  console.log(`Hello, my name is ${this.name}`);
}
```
The `this` keyword here refers to the scope it's being executed in. Since the `speak` function is within the scope of `kennan`, it refers to `kennan.name`.